packages/volto/src/config/index.js

```diff
@@ -149,6 +149,7 @@ let config = {
     serverConfig,
     storeExtenders: [],
     showTags: true,
+    showRelatedItems: true,
```

Comment:
There is still no suggestion for making it prettier, I suggest this feature is off by default making it a non-breaking feature that people can opt to enable and style as needed.

Reply:
I changed the setting, but I think it should be true by default, just like it is in the classic UI. This makes it even harder to test.